front/pages/w/[wId]/developers/api-keys.tsx

```diff
@@ -176,6 +176,23 @@ export function APIKeys({
                 use it to authenticate with the Dust API.
               </p>
               <br />
+              <div className="mt-4">
+                <Page.H variant="h5">Name</Page.H>
+                <Page.Horizontal align="center">
+                  <pre className="flex-grow overflow-x-auto rounded bg-muted-background p-2 font-mono dark:bg-muted-background-night">
+                    {keys[0]?.name}
+                  </pre>
+                  <IconButton
+                    tooltip="Copy to clipboard"
+                    icon={isCopiedDomain ? ClipboardCheckIcon : ClipboardIcon}
+                    onClick={async () => {
+                      if (keys[0]?.name) {
+                        await copyApiKey(keys[0].name);
```

Comment:
I think we should not use copyApiKey for everything because it can be like this:


https://github.com/user-attachments/assets/1513bd79-66ba-4ba8-bdc3-9b2bff7a1e17


We should have a proper way to copy for each field and show success state in the correct place 😄